radarr/naming.go

```diff
@@ -0,0 +1,66 @@
+package radarr
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"golift.io/starr"
+)
+
+// Define Base Path for Naming calls.
+const bpNaming = APIver + "/config/naming"
+
+// Naming represents the config/naming endpoint in Radarr.
+type Naming struct {
+	RenameMovies             bool   `json:"renameMovies,omitempty"`
+	ReplaceIllegalCharacters bool   `json:"replaceIllegalCharacters,omitempty"`
+	IncludeQuality           bool   `json:"includeQuality,omitempty"`
+	ReplaceSpaces            bool   `json:"replaceSpaces,omitempty"`
+	ID                       int64  `json:"id"` // ID must always be 1 (Oct 10, 2022)
+	ColonReplacementFormat   string `json:"colonReplacementFormat,omitempty"`
+	StandardMovieFormat      string `json:"standardMovieFormat,omitempty"`
+	MovieFolderFormat        string `json:"movieFolderFormat,omitempty"`
```

Comment:
I think these 2 cannot be empty

Reply:
Good point, I only tested with the weird `id` member. I'll do some more testing with smaller payloads.

Reply:
Indeed they are required.